test_scripts/Defects/5_0/1939_SDL_responds_with_wrong_result_in_case_receives_GetURLs_request_from_HMI_with_unknown_"service".lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1401
```

Comment:
@KrupenichOleg Description points to issue `1401` however branch name is `defect_1939`. Please correct.